bundles/org.openhab.automation.jsscripting/src/main/java/org/openhab/automation/jsscripting/internal/scriptengine/DelegatingScriptEngineWithInvocableAndCompilableAndAutocloseable.java

```diff
@@ -33,7 +33,7 @@ public abstract class DelegatingScriptEngineWithInvocableAndCompilableAndAutoclo
         implements ScriptEngine, Invocable, Compilable, AutoCloseable {
     protected T delegate;
 
-    public DelegatingScriptEngineWithInvocableAndCompilableAndAutocloseable(T delegate) {
+    protected DelegatingScriptEngineWithInvocableAndCompilableAndAutocloseable(T delegate) {
```

Comment:
I think this is not necessary.  It will never be used, except for setting to null.